### PR TITLE
fix: persisting browser url in heatmaps apge

### PR DESCRIPTION
**Pull request by pauldambra**
We persist the browser url for the in-app heatmaps page. 
This is intended to be helpful

If you swap projects or log into another account entirely. The in-app heatmaps page uses the now incorrect browser url because the persistence is not team/project aware

have seen this feedback a few different times.

### CODE REVIEW

File: frontend/src/scenes/heatmaps/heatmapsBrowserLogic.ts [hunk 1]
```diff
@@ -172,7 +175,7 @@ export const heatmapsBrowserLogic = kea<heatmapsBrowserLogicType>([
         ],
         browserUrl: [
             null as string | null,
-            { persist: true },
+            { persist: true, prefix: `team_${values.currentTeam?.id}` },
```
[the remark this answers is not in the document]
Comment (pauldambra) [reply frontend/src/scenes/heatmaps/heatmapsBrowserLogic.ts:178]: (the alternative for me here is not to persist this 🤷)

[the remark this answers is not in the document]
Comment (mariusandra) [reply frontend/src/scenes/heatmaps/heatmapsBrowserLogic.ts:178]: I mean there's this: https://keajs.org/docs/plugins/localstorage/#configuration-options -- definitely seems like worth hooking that up to `team.id`

File: frontend/src/index.tsx [hunk 2]
```diff
@@ -11,7 +11,9 @@ import { ErrorBoundary } from './layout/ErrorBoundary'
 import { loadPostHogJS } from './loadPostHogJS'
 
 loadPostHogJS()
-initKea()
+initKea({
+    prefix: `team_${window.POSTHOG_APP_CONTEXT?.current_team?.id}_project_${window.POSTHOG_APP_CONTEXT?.current_team?.project_id}`,
+})
```
Comment (pauldambra) [inline frontend/src/index.tsx:16]: we're always putting this info on the window so this will keep all team / project combos separate

Comment (Twixes) [reply frontend/src/index.tsx:16]: Doesn't this mean we also won't persist many user preferences cross-project? Like whether the sidebar is collapsed, some banners being acknowledged, replay speed, and such

**Review by mariusandra — APPROVED** [review]
should we do anything special to clear the data from localstorage once you log out? We may pollute something sensitive into `localStorage` otherwise 🤔

File: frontend/src/index.tsx (hunk 2 above)
Comment (pauldambra) [reply frontend/src/index.tsx:16]: it does... 

i think certainly we should include the team, maybe project is too extreme

**Comment by pauldambra** [comment]
> should we do anything special to clear the data from localstorage once you log out? We may pollute something sensitive into localStorage otherwise 🤔

Captain Scope Creep strikes again 🤣 

(we totally should, though)

**Review by Twixes — CHANGES_REQUESTED** [review]

File: frontend/src/index.tsx (hunk 2 above)
Comment (pauldambra) [reply frontend/src/index.tsx:16]: switched to a simpler thing :)

**Review by Twixes — APPROVED** [review]
Sorry, totally missed the re-review request! This approach is great.
(I pushed a commit to make this project-level rather than environment-level, but then realized that's actually dumb, since app URLs are most likely different between environments)

**Pull request merged**